packages/screens/Organizations/components/CreateDAOSection.tsx

```diff
@@ -49,6 +51,7 @@ export const CreateDAOSection: React.FC<CreateDAOSectionProps> = ({
   const selectedRadioStructure = watch("structure");
   const uri = watch("imageUrl");
   const name = watch("associatedHandle");
+  const isDev = process.env.NODE_ENV === "development";
```

Comment:
See https://github.com/TERITORI/teritori-dapp/blob/main/CONTRIBUTING.md#developer-mode
```suggestion
  const [isDev] = useDeveloperMode();
```